native/src/routes/settings/createSettingsSections.js

```diff
@@ -113,7 +125,21 @@ const createSettingsSections = ({ setSetting, t, languageCode, cityCode }: Creat
             throw Error('This error was thrown for testing purposes. Please ignore this error.')
           }
         }
-      }
+      },
+      ...(!buildConfig().featureFlags.jpalEvaluation
+        ? []
+        : [
+            {
+              accessibilityRole: 'none',
+              title: `JPAL ${t('evaluation')}`,
+              description: t('evaluationDescription'),
+              getSettingValue: (settings: SettingsType) => settings.trackingEnabled,
```

Comment:
I think this settings should be named `settings.jpalEnabled` as future users of the app will not necessarily want it enabled.

Reply:
Sorry @SteffiStuffel :D @maxammann is probably right. We should call it `jpalTrackingEnabled` though imo.